### ISSUE TITLE
about the result

### ISSUE THREAD

**Issue by demianzhang** [sunnyxiaohu/R-C3D.pytorch#7]
How about the result in R-C3D pytorch? 
Is it close to the original caffe3d code? 
I am training thumos14, however the loss is between 3 and 4, which dose not seem to decrease.
Thanks~

**Comment by sunnyxiaohu** [comment]
I trained a model based on the thumos14 dataset, the performance is almost the same or a little high than the author's.

**Comment by demianzhang** [comment]
Thanks, I will try again.

**Comment by rambleramble** [comment]
Can you comment on how the test results compare to Huijuan Xu's paper, Table 1 and 2 particularly？
https://arxiv.org/pdf/1703.07814.pdf

In terms of the testing performance, what would be the outcome of your test?
Thanks!
